internal/server/data/schema/parse.go

```diff
@@ -52,14 +58,14 @@ func ParseSchema(schema string) ([]Statement, error) {
 			if strings.HasPrefix(line, "CREATE SCHEMA") {
 				continue
 			}
-
 			var err error
 			if currentTable, line, err = parseStatementLine(line); err != nil {
 				return nil, err
 			}
 
 			currentStmt.WriteString("\n" + line + "\n")
-			if strings.HasSuffix(line, ";") {
+
+			if !inCodeBlock && strings.HasSuffix(line, ";") {
```

Comment:
If you got to the scanSchemaForStartOfStatement an `inCodeBlock` was set true, we should probably panic.